Hashing/Hashing.java

```diff
@@ -4,213 +4,187 @@
 	{
-		int flag=0;
-		int hashaddress=hashfunction(element);	//found hash address for element
-		
-		if(hashtable3[hashaddress]==-1)	//i.e. position currently empty
+	    hashtable3[hashaddress] = element;	//element inserted at appropriate position
+	}
+	else
+	{
+	    int lp = (hashaddress + 1) % 10;	//probing applied to look for empty space
+	    int i = 2;
+	    while (flag != 1)
+	    {
+	       	//found empty position?
+		if (hashtable3[lp] == -1)
 		{
-			hashtable3[hashaddress]=element;	//element inserted at appropriate position
+		    hashtable3[lp] = element;
+		    flag = 1;
 		}
-		
 		else
 		{
-			int lp=(hashaddress+1)%10;	//probing applied to look for empty space
-			int i=2;
-			
-			while(flag!=1)
-			{
-				//found empty position?
-				if(hashtable3[lp]==-1)
-				{
-					hashtable3[lp]=element;
-					flag=1;
-				}
-				
-				else
-				{
-					lp=(lp+(i^2))%10;	//else, keep looking quadratically incremented
-					i++;
-				}
-			}
+		    lp = (lp + (i ^ 2)) % 10;	//else, keep looking quadratically incremented
+		    i++;
 		}
+	    }
 	}
-	
-	//double hashing
-	void insertDH(int element)
+    }
+    //double hashing
+    void insertDH(int element)
+    {
+        int offset = 7 - (element % 7); 	//second hash function
+	int flag = 0;
+	int hashaddress = hashfunction(element);	//found hash address for element
+	if (hashtable4[hashaddress] == -1)	//i.e. position currently empty
 	{
-		int offset=7-(element%7); 	//second hash function
-		int flag=0;
-		int hashaddress=hashfunction(element);	//found hash address for element
-		
-		if(hashtable4[hashaddress]==-1)	//i.e. position currently empty
-		{
-			hashtable4[hashaddress]=element;	//element inserted at appropriate position
-		}
-		
-		else
+	    hashtable4[hashaddress] = element;	//element inserted at appropriate position
+	}
+	else
+	{
+	    while (flag != 1)
+	    {
+	        hashaddress = (hashaddress + offset) % 10;	//hash address changed
+		if (hashtable4[hashaddress] == -1)	//i.e. position currently empty
 		{
-			while(flag!=1)
-			{
-				hashaddress=(hashaddress+offset)%10;	//hash address changed
-				
-				if(hashtable4[hashaddress]==-1)	//i.e. position currently empty
-				{
-					hashtable4[hashaddress]=element;	//element inserted at appropriate position
-					flag=1;
-				}
-			}
+		    hashtable4[hashaddress] = element;	//element inserted at appropriate position
+		    flag = 1;
 		}
+            }
 	}
+    }
 	
-	//display elements in hashtable
-	void display(int n, int hashtable[])
+    //display elements in hashtable
+    void display(int n, int hashtable[])
+    {
+        System.out.println("\nIndex\tElement");
+	for (int i = 0; i< 10; i++)
 	{
-		System.out.println("\nIndex\tElement");
-		for(int i=0; i<10; i++)
-		{
-			if(hashtable[i]!=-1)
-			{
-				System.out.println(i+"\t"+hashtable[i]);
-			}
-		}
+            if (hashtable[i] != -1)
+	    {
+	        System.out.println(i + "\t" + hashtable[i]);
+	    }
 	}
+    }
 }
-
 /*
```

Comment:
Please add these headings to avoid confusion:
- `Input:` and `Output:` in this comment.
- The hashing method name when you print output in the console.

Reply:
Ok Ma'am, will do